### PR TITLE
Refactor scss

### PR DESCRIPTION
**Pull request by randallmorey**
Updates sass dependencies and moves rose into addon's app/styles folder.

Preparation for #1398

### CODE REVIEW

File: addons/rose/index.js [hunk 1]
```diff
@@ -9,38 +9,14 @@ module.exports = {
   included(app) {
     this._super.included.apply(this, arguments);
 
-    this.includeStyles(app);
+    app.import('node_modules/codemirror/lib/codemirror.css');
+    app.import('node_modules/codemirror/theme/monokai.css');
```
Comment (ZedLi) [inline addons/rose/index.js:13]: Nice, this is much cleaner!

**Review by ZedLi — APPROVED** [review]
LGTM but would love some context on this change

**Review by dvonanderson — APPROVED** [review]

File: addons/rose/index.js (hunk 1 above)
Comment (cameronperera) [reply addons/rose/index.js:13]: Does this remove the need to import these css files in our other `ember-cli-build` files as well?

**Review by cameronperera — APPROVED** [review]

File: addons/rose/index.js (hunk 1 above)
Comment (randallmorey) [reply addons/rose/index.js:13]: Yup.  Ideally Rose brings all the styles and assets along with it, with no additional configuration.  It's been a journey.

Comment (randallmorey) [reply addons/rose/index.js:13]: Note that this PR handles the removal of these imports from admin and desktop.

**Pull request merged**